src/main/java/net/sf/jabref/gui/groups/WarnAssignmentSideEffects.java

```diff
@@ -116,7 +59,7 @@ public static boolean warnAssignmentSideEffects(List<AbstractGroup> groups, Comp
         int choice = JOptionPane.showConfirmDialog(parent, message, Localization.lang("Warning"),
                 JOptionPane.YES_NO_OPTION, JOptionPane.WARNING_MESSAGE);
         return choice != JOptionPane.NO_OPTION;
-
+    
```

Comment:
I do not understand why they show up... If you see some of my other PRs
these spaces are actually removed...

I'll fix these things and merge.